Source/Entities/Validation/ValidationRuleInfo.cst

```diff
@@ -83,10 +83,18 @@ namespace <%=NameSpace%>.Validation
       /// <summary>
       /// Invokes the rule to validate the data.
       /// </summary>
+      /// <param name="target">The target object to validate.</param>
       /// <returns>True if the data is valid, False if the data is invalid.</returns>
-      public bool Invoke()
+      public bool Invoke(object target = null)

```

Comment:
Earlier version of C# do not support assigning a default value to a parameter like this.  For backwards compatibility, it would be better to write out both overloads and just have one call the other.

Reply:
Resolved